src/config/Settings.ts

```diff
@@ -0,0 +1,98 @@
+import { Feature, FeatureFlag } from './Feature';
```

Comment:
Huh - my comment about this got lost from the code review...

Can there be a test to exercise the code in Feature feature please?